frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineDetails.tsx

```diff
@@ -45,6 +46,7 @@ const PipelineDetails: React.FC<PipelineDetailsProps> = ({
             links={taskLinks}
             title={t('pipelines-plugin~Tasks')}
           />
+          <WorkspaceDefinitionList workspaces={pipeline?.spec.workspaces} />
```

Comment:
spec is not optional but workspaces is
```suggestion
          <WorkspaceDefinitionList workspaces={pipeline.spec?.workspaces} />
```